config.json.demo

```diff
@@ -1,11 +1,8 @@
 {  "accounts":
     [
-        {"id": "123456789012", "name": "demo", "default": true}
+        {"id": "820411415250", "name": "Helix Master", "default": true}
```

Comment:
This change should be revert to the original config.json.demo file.  You should use `config.json` for your config file.